src/app/locales/zh-cn/models.yml

```diff
@@ -0,0 +1,21 @@
+twitch:
+    user:
+        view_count: |
+            {count, plural,
+                one {观看次数: 1 }
+                other {观看次数: #}
+            }
+    stream:
+        started-at:
+            less-than-24h: "于 {started_at, time, medium} 开始"
+            more-than-24h: "于 {started_at, time, long} 开始"
+        viewer_count: |
+            {count, plural,
+                one { 1 人正在观看}
+                other { # 人正在观看}
+            }
```

Comment:
Is it intentional that the same translation gets used for the different quantities? According to DeepL.com, "人正在观看" is plural, but it also gets used for the `one` quantity. Since I only speak German and English, all I can do is ask.

I noticed the same on other plural translations. Remember that the plural syntax allows for precise translations depending on the quantity (and grammar of the language).

https://github.com/streamlink/streamlink-twitch-gui/wiki/Translating#pluralization

Reply:
In Chinese, there is usually no distinction between singular and plural forms of verbs. Basically, it is only necessary to change the subject's singular or plural form. Based on my own experience using Chinese, this is not a problem.

Reply:
So the `one` quantity could be removed then as it's redundant.
https://unicode-org.github.io/cldr-staging/charts/latest/supplemental/language_plural_rules.html#zh

Reply:
Fixed.